THCalendarDatePicker/THDateDay.m

```diff
@@ -88,9 +88,7 @@ - (void)setCurrentColors {
 
 -(void)setEnabled:(BOOL)enabled {
     [self.dateButton setEnabled:enabled];
-    if (!enabled) {
```

Comment:
This check is important, otherwise the days that are shown, but not in the current month are not handled correctly